pages/conference/_eventType/_id.vue

```diff
@@ -327,6 +327,7 @@ export default {
 }
 .speech__tabParagraphTitle {
     @apply font-serif font-bold mb-2 mr-8 text-pink-700;
+    min-width: 85px;
```

Comment:
the titles are not aligned without this line (as shown below), but it's merely a temporary solution. 
@furecool please help resolve this issue in the following PR.

<img width="510" alt="image" src="https://user-images.githubusercontent.com/24987826/178154685-dc4b8c84-ef5a-40e9-8ef7-c1cb9fa4d4dd.png">

Reply:
Oh no, I didn't notice the problem before.
Got it.